### ISSUE TITLE
Node Deletion

### ISSUE THREAD

**Issue by bdwyertech** [wazuh/wazuh-api#19]
Seeing something weird after upgrading to 2.1.0 -- seems like the delete endpoint is breaking ID's into individual strings, quite weird.

```bash
curl -X DELETE http://abc.123.net:55000/agents/208 -H 'authorization: Basic ABC123'
```

```json
{
    "error": 0,
    "data": {
        "msg": "Some agents were not removed",
        "ids": [
            "2",
            "0",
            "8"
        ]
    }
}
```

**Comment by mgmacias95** [comment]
Hi @bdwyertech,

This is a known issue that has been fixed in commit 167db5b4cc20e13818f2243ec815684233d97274. This bug is going to be included in next release packages which is going to be soon. If you want to have this bug fixed, you could install the API from sources. Note that you must install branch `2.1` and not `master`.

Best regards and thank you for your contribution,
Marta

**Comment by bdwyertech** [comment]
Ok, thanks for that info.

Brian
